### ISSUE TITLE
Test: Multi-level hierarchy of tests requires Fact attributes to be specified again

### ISSUE THREAD

**Issue by smitpatel** [aspnet/EntityFrameworkCore#8050]
Those tests can be made part of middle layer from `QueryTestBase` to provider specific QueryTest.

It creates 3 level hierarchy which is blocked on https://github.com/xunit/xunit/issues/1068

**Comment by smitpatel** [comment]
RelationalQueryTestBase was removed in #8084 .
This is no longer needed.

**Comment by smitpatel** [comment]
We are hitting this issue in `ComplexNavigationOwnedQueryTestBase` now. Looking at overriding our test discoverer to make it work & remove it once xunit fixes the issue.

**Comment by smitpatel** [comment]
Removing from backlog. This cannot be fixed without upgrading our specs projects to xunit beta4 or higher but specs are stuck on 2.2. We need to follow up with aspnet team on this.
